src/components/toast/toast.style.ts

```diff
@@ -79,17 +75,66 @@ StyledPortal.defaultProps = {
 
 const animationName = ".toast";
 const alternativeAnimationName = ".toast-alternative";
-const ToastStyle = styled(MessageStyle)<{
+
+const ToastColourVariants = {
+  error: "var(--colorsSemanticNegative500)",
+  info: "var(--colorsSemanticInfo500)",
+  success: "var(--colorsSemanticPositive500)",
+  warning: "var(--colorsSemanticCaution500)",
+  neutral: "var(--colorsSemanticNeutral500)",
+};
+
+type ToastVariants = "error" | "info" | "success" | "warning" | "neutral";
```

Comment:
Comment (non-blocking): we're already exporting `MessageVariants` as a type, I think it would be fine to re-use that here as it's not really coupling imo

Reply:
I can get that done. If they diverge even more in the future, we can always define a new type here like this.